src/process/sources/index.html

```diff
@@ -34,9 +34,17 @@ <h2 class="content-l_col-1-4 content-l_col">
                         <div class="sources-step">
                             <h3>Assess your spending</h3>
                             <h4>&ldquo;About half of Americans don’t have a budget.&rdquo;</h4>
-                            <p>Center for Financial Services Innovation. (March 2015). <i>Understanding and Improving Consumer Financial Health in America</i>, p. 9.  Available at: <a href="http://www.cfsinnovation.com/Document-Library/Understanding-Consumer-Financial-Health">http://www.cfsinnovation.com/Document-Library/Understanding-Consumer-Financial-Health</a>.</p>
+                            <p>Center for Financial Services Innovation. (March 2015). <i>Understanding and Improving Consumer Financial Health in America</i>, p. 9.  Available at: 
+                                <a class="icon-link icon-link__external-link" href="http://www.cfsinnovation.com/Document-Library/Understanding-Consumer-Financial-Health">
```

Comment:
Do we want to add the `target="_blank"` attribute so these  links open in a separate tab/window?